great_expectations/rule_based_profiler/expectation_configuration_builder/expectation_configuration_builder.py

```diff
@@ -10,12 +10,12 @@
 
 
 class ExpectationConfigurationBuilder(Builder, ABC):
-    def __init__(
-        self,
-        expectation_type: str,
-    ):
+    def __init__(self, expectation_type: str, **kwargs):
         self._expectation_type = expectation_type
 
+        for k, v in kwargs.items():
+            setattr(self, k, v)
```

Comment:
Let's add a logging statement to any setattr behavior? Akin to what we have in `config/base.py`?

Reply:
@cdkini Adding it (in next commit) -- this is very important.  Thanks.

Reply:
@cdkini I added a comment, but not logging -- these warnings seem to be very voluminous.  I feel that we should either make them happen only once (not sure how), or add a comment instead.